migration/v0.45/prepare/40-preserve-current-os-setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+  log_info "Moving current Opensearch configuration to override config"
+  yq_add sc .opensearch "$(yq_dig sc .opensearch | yq4 -o json)"
```

Comment:
https://github.com/elastisys/compliantkubernetes-apps/blob/4d3deefb5b31e701d9bc20338f3dedab2a646792/scripts/migration/yq.sh#L4-L5
And `opensearch`

If you want to copy the entire `opensearch` key, then use `yq_merge` as the source, else run `yq_dig` on individual options you want to keep that aren't a map.

Reply:
Okay, I guess I missed that.

This is how you meant right? https://github.com/elastisys/compliantkubernetes-apps/pull/2444/commits/3f5c4c8903e846bd5ca0c3aa68febd8afae235d4